src/new-client/src/App/Analytics/AnalyticsCore.tsx

```diff
@@ -7,20 +7,71 @@ import {
   AnalyticsStyle,
   AnalyticsHeader,
   AnalyticsInnerWrapper,
+  RightNav,
+  HeaderAction,
 } from "./styled"
 import { createSuspenseOnStale } from "@/utils/createSuspenseOnStale"
 import { isAnalyticsDataStale$ } from "@/services/analytics"
 
+import { PopOutIcon } from "@/components/icons/PopOutIcon"
+import styled from "styled-components"
+import { tearOutAnalytics, tearOutEntryAnalytics$ } from "@/Web/tearoutSections"
+import { useObservableSubscription } from "@/utils/useObservableSubscription"
+import { openWindow } from "@/Web/utils/window"
+import { constructUrl } from "@/utils/url"
+
+import {
+  useTearOutTileState,
+  useTearOutTradeState,
+} from "@/Web/tearoutSections"
+
 const analytics$ = merge(pnL$, profitAndLoss$, positions$)
 
 const SuspenseOnStaleData = createSuspenseOnStale(isAnalyticsDataStale$)
 
+var isTornOut = false
 const Analytics: React.FC = ({ children }) => {
+  const tearOutTradeState = useTearOutTradeState("Trade")
```

Comment:
Similar logic is used in the other components. Perhaps a util function somewhere would be better.
eg:
const canBeTornOut = (windowToTearOut) => yes or no based on the state of tear outs